packages/ssx-sdk/src/modules/Credential.ts

```diff
@@ -0,0 +1,19 @@
+import { IDataVault } from './Storage';
+
+interface ICredential {
```

Comment:
I'm not sure exactly how this will work, like the CRUD stuff seems like a DataStore concern, and I'm not sure coupling the credential to it's storage does much other than add coupling. I'd think that the things this module would do would be like VPs or credential validation. 

Doesn't matter at the moment though, this is just the rough sketch.